tests/functional/cylc-combination-scripts/vr_workflow_stop/flow.cylc

```diff
@@ -0,0 +1,8 @@
+[scheduling]
+    initial cycle point = 1500
+    [[graph]]
+        P1Y = foo
+
+[runtime]
+    [[foo]]
+        script = cylc stop --now --now ${CYLC_WORKFLOW_ID}
```

Comment:
A sneaky way to do this more reliably is to set the "startup" handler to run the command "cylc stop" that way the workflow will shutdown straight away for both initial and restarts.